etc/profile-m-z/parsecd.profile

```diff
@@ -0,0 +1,38 @@
+
+# Firejail profile for Parsec
+# Description: Remote desktop application focused on gaming and other 3D applications
+# This file is overwritten after every install/update
+
+# Persistent local customizations
+include parsecd.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.parsec
```

Comment:
What about `include disable-*.inc`?

Reply:
There are only some minor json configs, binary user and device IDs and the .so thing I mentioned below in that dir. I do not think the blacklists will do much good here because there is simply nothing there. Do you have a specific thing in mind here?

Reply:
I don't talk about the noblacklist, just needed to put the comment somewhere.

```
ignore noexec ${HOME}

include disable-common.inc
include disable-devel.inc
include disable-exec.inc
include disable-interpreters.inc
include disable-proc.inc
include disable-programs.inc
include disable-shell.inc
include disable-xdg.inc
```